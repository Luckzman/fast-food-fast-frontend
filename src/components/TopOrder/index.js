import React from 'react';
import { Link } from 'react-router-dom';
import './topOrder.scss';

const TopOrder = () => (
  <div className="top-order">
    <div className="container">
      <h1>top order</h1>
      <p>Customer's Top Most Choice</p>
      <div className="order-row">
        <div className="order-card">
          <div className="order-img">
            <img src="https://res.cloudinary.com/dx0nauane/image/upload/v1552050529/image-4.jpg" alt="" />
            <Link to="./html/order-details.html">view details</Link>
          </div>
          <div className="card-content">
            <h3>meat</h3>
            <div className="star-rating">
              <span className="fa fa-star checked" />
              <span className="fa fa-star checked" />
              <span className="fa fa-star checked" />
              <span className="fa fa-star checked" />
              <span className="fa fa-star checked" />
            </div>
            <p>Very testy and delicious with nice blend of fried plantain</p>
          </div>
        </div>
        <div className="order-card">
          <div className="order-img">
            <img src="https://res.cloudinary.com/dx0nauane/image/upload/v1552050505/image-12.jpg" alt="" />
            <a href="./html/order-details.html">view details</a>
          </div>
          <div className="card-content">
            <h3>plantain sauce</h3>
            <div className="star-rating">
              <span className="fa fa-star checked" />
              <span className="fa fa-star checked" />
              <span className="fa fa-star checked" />
              <span className="fa fa-star checked" />
              <span className="fa fa-star checked" />
            </div>
            <p>Very testy and delicious with nice blend of fried plantain</p>
          </div>
        </div>
        <div className="order-card">
          <div className="order-img">
            <img src="https://res.cloudinary.com/dx0nauane/image/upload/v1544280158/x5cu9owt6sx4vxhdoz23.jpg" alt="" />
            <a href="./html/order-details.html">view details</a>
          </div>
          <div className="card-content">
            <h3>meat</h3>
            <div className="star-rating">
              <span className="fa fa-star checked" />
              <span className="fa fa-star checked" />
              <span className="fa fa-star checked" />
              <span className="fa fa-star checked" />
              <span className="fa fa-star unchecked" />
            </div>
            <p>Very testy and delicious with nice blend of fried plantain</p>
          </div>
        </div>
      </div>
    </div>
  </div>
);

export default TopOrder;
