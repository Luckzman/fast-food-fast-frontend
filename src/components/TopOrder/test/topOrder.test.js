import React from 'react';
import { shallow } from 'enzyme';
import TopOrder from '..';

describe('App', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<TopOrder />);
    expect(wrapper).toMatchSnapshot();
  });
});
