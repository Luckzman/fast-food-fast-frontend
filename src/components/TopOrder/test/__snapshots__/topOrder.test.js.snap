// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<div
  className="top-order"
>
  <div
    className="container"
  >
    <h1>
      top order
    </h1>
    <p>
      Customer's Top Most Choice
    </p>
    <div
      className="order-row"
    >
      <div
        className="order-card"
      >
        <div
          className="order-img"
        >
          <img
            alt=""
            src="https://res.cloudinary.com/dx0nauane/image/upload/v1552050529/image-4.jpg"
          />
          <Link
            replace={false}
            to=""
          >
            view details
          </Link>
        </div>
        <div
          className="card-content"
        >
          <h3>
            meat
          </h3>
          <div
            className="star-rating"
          >
            <span
              className="fa fa-star checked"
            />
            <span
              className="fa fa-star checked"
            />
            <span
              className="fa fa-star checked"
            />
            <span
              className="fa fa-star checked"
            />
            <span
              className="fa fa-star checked"
            />
          </div>
          <p>
            Very testy and delicious with nice blend of fried plantain
          </p>
        </div>
      </div>
      <div
        className="order-card"
      >
        <div
          className="order-img"
        >
          <img
            alt=""
            src="https://res.cloudinary.com/dx0nauane/image/upload/v1552050505/image-12.jpg"
          />
          <Link
            replace={false}
            to=""
          >
            view details
          </Link>
        </div>
        <div
          className="card-content"
        >
          <h3>
            plantain sauce
          </h3>
          <div
            className="star-rating"
          >
            <span
              className="fa fa-star checked"
            />
            <span
              className="fa fa-star checked"
            />
            <span
              className="fa fa-star checked"
            />
            <span
              className="fa fa-star checked"
            />
            <span
              className="fa fa-star checked"
            />
          </div>
          <p>
            Very testy and delicious with nice blend of fried plantain
          </p>
        </div>
      </div>
      <div
        className="order-card"
      >
        <div
          className="order-img"
        >
          <img
            alt=""
            src="https://res.cloudinary.com/dx0nauane/image/upload/v1544280158/x5cu9owt6sx4vxhdoz23.jpg"
          />
          <Link
            replace={false}
            to=""
          >
            view details
          </Link>
        </div>
        <div
          className="card-content"
        >
          <h3>
            meat
          </h3>
          <div
            className="star-rating"
          >
            <span
              className="fa fa-star checked"
            />
            <span
              className="fa fa-star checked"
            />
            <span
              className="fa fa-star checked"
            />
            <span
              className="fa fa-star checked"
            />
            <span
              className="fa fa-star unchecked"
            />
          </div>
          <p>
            Very testy and delicious with nice blend of fried plantain
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
